voluncheer/templates/base.html

```diff
@@ -7,7 +7,7 @@
     <title>Voluncheer</title>
     <link rel="stylesheet" href="{% static 'css/app.css' %}">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">
-    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
```

Comment:
Why are we changing jquery version here?

Reply:
I changed it during debugging to see if it would help. Changing it did resolve a persistent error I see when looking at the Chrome debugging console (something about tooltip) but I don't believe the newer version is necessary to support these changes so I'll revet it.